data/blog/bitcoin-grants-july-2023.mdx

```diff
@@ -0,0 +1,265 @@
+
+Repository: [zerosync/zerosync](https://github.com/zerosync/zerosync)  
+License: MIT
+
+### Mutiny Wallet
+
+Mutiny Wallet is a web-first wallet capable of running anywhere, providing
+instant onboarding and platform censorship resistance. It is self-custodial,
+privacy-focused, user-friendly, and open-sourced under the MIT license.
+
+The wallet has a strong focus on privacy, scalability, and accessibility. In
+addition to features that you would expect a regular lightning wallet to have,
+the team is working to incorporate Nostr-related features into the wallet, such
+as a feed of friends' Zaps, native Zap sending and receiving, a lightning
+subscription specification for services such as nostr relays, and a P2P DLC
+marketplace. The team's goal is to provide users with a seamless experience,
+combining the power of Bitcoin and Lightning with social media in a way that
+matches the Bitcoin ethos.
+
+Repository: [MutinyWallet](https://github.com/MutinyWallet)  
+License: MIT
+
+### next-auth Lightning Provider
+
+The goal of this project is to implement an authentication provider for
+next-auth, an authentication provider for the popular open-source framework
+NextJS. The next-auth framework has nearly 500k weekly downloads and powers the
+authentication of many modern web, mobile, and desktop apps. Having a
+plug-and-play Provider for Lightning makes integration easier and more
+attractive for developers.
+
+Repository:
+[jowo-io/next-auth-lightning-provider](https://github.com/jowo-io/next-auth-lightning-provider)  
+License: ISC
+
+### Cashu
+
+Cashu is a Chaumian ecash system built for bitcoin that brings near-perfect
+privacy for users of custodial bitcoin applications. A Cashu ecash mint does not
+know who you are, what your balance is, or who you're transacting with. Users of
+a mint can exchange ecash privately, without anyone being able to know who the
+involved parties are.
+
+Payments are executed without anyone able to censor specific users. There are
+multiple implementations of the Cashu protocol. Popular open-source wallets are
+Cashu Nutshell, Cashu.me, and Nutstash.
+
+Repository: [cashubtc/cashu](https://github.com/cashubtc/cashu)  
+License: MIT
+
+### lnproxy
+
+lnproxy is a simple privacy tool that empowers users of custodial Lightning
+wallets with better payment destination privacy and sovereign node runners with
+enhanced receiver privacy. lnproxy works like a "poor man's" rendezvous router,
+providing privacy for users without taking custody of their funds. The project
+encompasses an LNURL-style protocol specification and a collection of
+open-source implementations of lnproxy clients and a relay.
+
+Repository: [lnproxy/lnproxy](https://github.com/lnproxy/lnproxy)  
+License: GPL 3.0 & MIT
+
+### Blixt Wallet
+
+Blixt is a non-custodial Lightning wallet for Android, iOS, and macOS with a
+focus on usability and user experience. The project's current target audience is
```

Comment:
I would start this paragraph off with "Blixt is a non-custodial wallet whose current audience is bitcoiners who want to give Lightning a try". I feel like that draws those potential people in more, and then you can give them details that they may not understand.